packages/app-store/slackapp/api/interactiveHandler.ts

```diff
@@ -0,0 +1,24 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { parse } from "querystring";
+
+import createEvent from "../lib/actions/createEvent";
+
+enum InteractionEvents {
+  CREATE_EVENT = "cal.event.create",
+}
+
+export default async function interactiveHandler(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
Handle interactions - any button clicks in slack that require processing to happen